pkg/testing/fixture_install.go

```diff
@@ -322,6 +418,30 @@ func (i UninstallOpts) toCmdArgs() []string {
 
 // Uninstall uninstalls the installed Elastic Agent binary
 func (f *Fixture) Uninstall(ctx context.Context, uninstallOpts *UninstallOpts, opts ...process.CmdOption) ([]byte, error) {
+	switch f.packageFormat {
+	case "targz", "zip":
+		return f.uninstallNoPkgManager(ctx, uninstallOpts, opts)
+	case "deb":
+		return f.uninstallDeb(ctx, uninstallOpts, opts)
+	default:
+		return nil, fmt.Errorf("uninstall of package format '%s' not supported yet", f.packageFormat)
+	}
+}
+
+func (f *Fixture) uninstallDeb(ctx context.Context, uninstallOpts *UninstallOpts, opts []process.CmdOption) ([]byte, error) {
+	// stop elastic-agent, non fatal if error, might have been stopped before this.
+	out, err := exec.CommandContext(ctx, "sudo", "systemctl", "stop", "elastic-agent").CombinedOutput()
```

Comment:
Is making the deb smart enough to start/stop the service a possibility? 

Not something for this PR, just wondering if this another point of friction we've arbitrarily introduced to using these packages.

Reply:
+1